USWDS 3.0 - Package page edits (#1560)
uswds/uswds-site: pages/ui-components/packages.md

@forward "uswds-helpers";
```

If you suffer from slow compiles, it may be worth experimenting with source forwarding. 

## Package Sass requirements: Load paths

> Using USWDS 3.0 and packages requires compiling your Sass with load paths. Load paths tell your Sass compiler where to look for USWDS packages. Any compiler needs to include the following load paths:

Reviewer: ```suggestion
Using USWDS 3.0 and packages requires compiling your Sass with load paths. Load paths tell your Sass compiler where to look for USWDS packages. Any compiler needs to include a load path to the `@uswds/uswds/packages` directory:
```

Author: Updated in b05e3d4baddb9d5940f7ee911009897b7410b6b0

```js

> "./node_modules/@uswds/uswds",

Author: Flagging because it is that time of day and I need a sanity check on these include paths

Reviewer: I think I may remove this first reference. It _is_ intended to reference the `@uswds` directory in order to support — down the road — component packages published into that org scope... like `@uswds/usa-banner`. But this functionality is not here yet, so it's probably more confusing than it needs to be

Reviewer: ```suggestion
```

Author: Removed in b05e3d4baddb9d5940f7ee911009897b7410b6b0

"./node_modules/@uswds/uswds/packages"
```

In a gulpfile, use `includePaths` in your `sass()` function.
```js
  sass({
    includePaths: [

>       "./node_modules/@uswds/uswds",

Author: Flagging because it is that time of day and I need a sanity check on this one.

>       "./node_modules/@uswds/uswds",

Reviewer: ```suggestion
```

Author: Removed in b05e3d4baddb9d5940f7ee911009897b7410b6b0

```js
loader: "sass-loader",
options: {
  sassOptions: {
    includePaths: [

>       "./node_modules/@uswds/uswds",

Reviewer: ```suggestion
```

Author: Removed in b05e3d4baddb9d5940f7ee911009897b7410b6b0
